app/assets/stylesheets/header.scss

```diff
@@ -118,31 +118,107 @@
       }
     }
   }
-  [class^="entypo-"], [class*="entypo-"] {
-    color: inherit;
-    font-size: $font-size-h3;
-    vertical-align: middle;
-  }
-  .nav-badge {
-    margin-bottom: -2px;
-    .badge {
-      position: absolute;
-      right: 10px;
-      top: 10px;
-    }
-  }
 
-  #user_menu {
+  .user-avatar {
+    height: $navbar-height;
+    margin-bottom: -$navbar-padding-vertical;
+    margin-right: 10px;
+    margin-top: -$navbar-padding-vertical;
+    padding: ($navbar-height - 30px) / 2 0;
+
     .avatar {
       height: 30px;
       width: 30px;
     }
-    .user-avatar {
-      height: $navbar-height;
-      padding: ($navbar-height - 30px)/2 0;
-      margin-bottom: -$navbar-padding-vertical;
-      margin-top: -$navbar-padding-vertical;
-      margin-right: 10px;
+  }
+
+  @media (max-width: $grid-float-breakpoint-max) {
+
+    .navbar-nav.hidden-xs {
+      margin: 0;
+      margin-top: 6px;
+    }
+
+    .nav-badge {
+      color: $navbar-inverse-link-color;
+      &:hover { color: $navbar-inverse-link-hover-color; }
+      &:hover,
+      &:focus {
+        background-color: transparent;
+      }
+    }
+
+    .form-group,
+    .twitter-typeahead {
+      margin-bottom: 0;
+
+      &,
+      input { width: 100%; }
+    }
+
+    .nav-badges .dropdown-menu {
+      width: 300px;
+    }
+
+    .user-menu-dropdown {
+      background-color: transparent;
+      border: 0;
+      box-shadow: none;
+      display: block;
+      margin-top: -8px; // To compensate parent ul margin
+      padding: 0;
+      position: static;
+      width: 100%;
+
+      a {
+        color: #9d9d9d;
```

Comment:
You have the variables [here](https://github.com/diaspora/diaspora/blob/develop/app/assets/stylesheets/bootstrap-variables.scss) and you can search if you find it there. But that's probably not what you are looking for. When you know where this color is used in bootstrap, inspect the element in the browser, so you find the generated css from bootstrap in the browser. In the generated desktop.css there are comments above each rule, something like `/home/benjamin/.rvm/gems/ruby-2.4.0@diaspora/gems/bootstrap-sass-3.3.7/assets/stylesheets/bootstrap/...` then open that file and see which variable is used. Maybe there is an easier way, but I don't know it, that's what I did when I created the dark theme to find the variables I need to change.